db/config.cc

```diff
@@ -38,6 +38,9 @@
 #include <seastar/core/metrics_api.hh>
 #include <seastar/core/relabel_config.hh>
 #include <seastar/util/file.hh>
+#include "init.hh"
+
+static logging::logger clog("config");
```

Comment:
unused?

Reply:
left over from an intermediate version.  will cleanup

Reply:
dropped in v5